praxis-isncsci/ui
# Fixed inputs and imports

Closes #175 

## Problem

The current **rollup** configuration is generating several `index` files because imports are missing to be added to the **Rollup.config** file.

## Solution

- Add `src/core/helpers/index.ts` to the **Rollup.config` inputs list
- Import domain, boundaries, use cases and helpers straight from the index files
- Fix imports from `isncsci/cjs/interfaces`

## Testing

- [x] All required inputs and imports have been added

<details>
  <summary>Test case</summary>

  1. Run `npm run build`
  2. Confirm that no `index.js` files are created at the root of the project

</details>
